src/ui/field_formats_mixin.js

```diff
@@ -0,0 +1,7 @@
+import { FieldFormatsService } from '../core_plugins/kibana/common/field_formats/field_formats';
+import { getUiSettingDefaults } from '../core_plugins/kibana/ui_setting_defaults';
+
+export function fieldFormatsMixin(kbnServer, server) {
+  server.decorate('server', 'uiSettingDefaults', getUiSettingDefaults());
```

Comment:
uiSettings.getDefaults() is just a wrapper around options.getDefaults. If options.getDefaults is not provided than the default implementation of '() => ({})' is used. I was unable to figure out how to create a functioning getDefaults implementation to pass to `server.uiSettingsServiceFactory` that would return the defaults. Please advise how to get this to work.

If field_formats_mixin is not the place, then where should uiSettingDefaults be exposed to the server?